Rubberduck.Parsing/Symbols/Attributes.cs

```diff
@@ -148,8 +141,8 @@ public void AddHiddenMemberAttribute(string identifierName)
 
         public bool HasHiddenMemberAttribute(string identifierName, out AttributeNode attribute)
         {
-            attribute = this.SingleOrDefault(a => a.HasValue("-4") &&
-                a.Name.Equals($"{identifierName}.VB_UserMemId", StringComparison.OrdinalIgnoreCase));
+            attribute = this.SingleOrDefault(a => a.HasValue("40")
+                && a.Name.Equals($"{identifierName}.VB_UserMemId", StringComparison.OrdinalIgnoreCase));
```

Comment:
This is incorrect. The actual generated attribute for hidden member is:

`Attribute <procedure>.VB_MemberFlags = "40"`

`VB_UserMemId` corresponds to the DISPID attribute, and a positive number cannot be used for meta with that attribute.

Reply:
As mentioned in [chat](), these are all members of the `[FUNCFLAGS]`(https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.typelibfuncflags?view=netframework-4.7.2) enum. I would assume that since the attributes are being used to generate the internal TypeLib, that would be a good candidate enum to use. It might be useful in integrating this down the road.

Reply:
I will fix that in a future PR, not this one.